### ISSUE TITLE
Can't load url.URL variable

### ISSUE THREAD

**Issue by prochac** [Showmax/env#4]
If I try to load `url.URL` variable then I get no error but variable isn't loaded.
example here: https://play.golang.org/p/ecdDsQLWN4X

`url.URL` should be supported by README
https://github.com/Showmax/env#list-of-default-parsers

The broken "github.com/kelseyhightower/envconfig" works fine here: https://play.golang.org/p/lywLmgyGDc1

**Comment by prochac** [comment]
Here is a git diff for env_test.go file to see the problem.
```
diff --git a/env_test.go b/env_test.go
index c00d14e..2c8c9a3 100644
--- a/env_test.go
+++ b/env_test.go
@@ -3,6 +3,7 @@ package env
 import (
 	"fmt"
 	"math"
+	"net/url"
 	"os"
 	"strconv"
 	"strings"
@@ -24,6 +25,8 @@ type config struct {
 	Foo
 	Bar         `env:"BAR_"`
 	Bool        *bool         `env:"BOOL"`
+	URL         url.URL       `env:"URL"`
+	URLPtr      *url.URL      `env:"URL_PTR"`
 	Duration    time.Duration `env:"DURATION"`
 	Inner       Foo           `env:"INNER_"`
 	Int         int           `env:"INT"`
@@ -56,6 +59,8 @@ var (
 		"BAR_BAR":      "BAR_BAR",
 		"FOO":          "FOO",
 		"BOOL":         "true",
+		"URL":          "https://example.org",
+		"URL_PTR":      "https://example.org",
 		"DURATION":     "10ms",
 		"INNER_FOO":    "INNER_FOO",
 		"INT":          "1",
@@ -75,6 +80,8 @@ var (
 		Foo:         Foo{"FOO"},
 		Bar:         Bar{"BAR_BAR"},
 		Bool:        &trueVar,
+		URL:         url.URL{Scheme: "https", Host: "example.org"},
+		URLPtr:      &url.URL{Scheme: "https", Host: "example.org"},
 		Duration:    10 * time.Millisecond,
 		Inner:       Foo{"INNER_FOO"},
 		Int:         1,
```

**Comment by jan-dubsky** [comment]
Closed by #5.

To sum up what happened. `url.URL` is a structure. It doesn't implement `TextUnmarshaller`, so it was not handled by automatic parsing mechanism. And there was a missing parser specifically for `url.URL`. There was only the `*url.URL` parser, which obviously didn't match. Consequently, the `env` package followed it's struct parsing strategy and recursively parsed all fields of `url.URL` structure. Obviously none of those fields had `env` tag. Consequently, from the point of `env` package, the parsing was correctly done, but the `url.URL` remained untouched.

**Comment by prochac** [comment]
I have to import it this way:
```
github.com/Showmax/env v1.0.2-0.20201215172346-ee8163db7c74
```
